www/includes/utility.php

```diff
@@ -596,11 +586,8 @@ function htmlentities_notags($text) {
     $text = strtr($text, $tbl);
 
     // This turns any out of bounds characters like fancy quotes
-    // into the ISO-8859-1 equivalent if possible.
-    $text = iconv('Windows-1252', 'ISO-8859-1//TRANSLIT', $text);
-
-    # Remove all illegal HTML characters (damn you, Windows-1252)
-    $text = preg_replace('/[\x80-\x9f]/', '', $text);
+    // into the UTF-8 equivalent if possible.
+    $text = iconv('Windows-1252', 'UTF-8', $text);
```

Comment:
This will break UTF-8 text, because UTF-8 uses bytes from the Windows-1252 128-159 range. e.g. It would turn é (c3a9) into Ã© (c383c2a9). I don't think you need any iconv here at all.